common/go-utils/src/bin/diff_inventory.rs

```diff
@@ -39,7 +39,8 @@ fn main() {
     .filter(|(_, artifact_diff)| !artifact_diff.is_empty())
     .for_each(|(action, artifacts)| {
         let mut list: Vec<&Artifact<GoVersion>> = artifacts.iter().collect();
-        list.sort_by_key(|a| &a.version);
+        list.sort();
+        list.reverse();
```

Comment:
Does this mean the default sorting is version ascending (i.e.; oldest to newest)? Shouldn't it be the opposite so you don't need to `reverse()` it?